app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.jsx

```diff
@@ -0,0 +1,81 @@
+/* @flow */
+/*eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+
+import React from 'react'
+import classnames from 'classnames'
+
+import {
+  Avatar
+} from '../'
+
+type MultipleUsersStackedProps = {
+  className?: String,
+  id?: String,
+  users: Array<Object>,
+}
+
+const MultipleUsersStacked = ({
+    className,
+    id,
+    users
+  }: MultipleUsersStackedProps) => {
+
+  const multipleUsersStackedCss = () => {
```

Comment:
This doesn't have to be a function either, and can also be inlined in the single usage on line 73